src/OrleansRuntime/GrainDirectory/MultiClusterRegistration/GlobalSingleInstanceRegistrar.cs

```diff
@@ -95,29 +95,27 @@ public void Delete(GrainId gid)
                 if (logger.IsVerbose)
                     logger.Verbose("GSIP:Req {0} Round={1} Act={2}", address.Grain.ToString(), numRetries - retries, myActivation.Address.ToString());
 
-                var responses = SendRequestRound(address, remoteClusters);
-
-                var outcome = await responses.Task;
+                var outcome = await SendRequestRound(address, remoteClusters);
 
                 if (logger.IsVerbose)
-                    logger.Verbose("GSIP:End {0} Round={1} Outcome={2}", address.Grain.ToString(), numRetries - retries, responses);
+                    logger.Verbose("GSIP:End {0} Round={1} Outcome={2}", address.Grain.ToString(), numRetries - retries, outcome.State);
```

Comment:
Did you mean to make the tracing message less informative? It no longer prints the activation address if a remote owner is found. Before, this called a custom ToString() method on the response tracker that included the activation address.